packages/cli/test/integration.test.ts

```diff
@@ -89,7 +122,7 @@ const waitForDeployment = async href => {
   }
 };
 
-function fetchTokenInformation(token, retries = 3) {
+function fetchTokenInformation(token: any, retries = 3) {
```

Comment:
```suggestion
function fetchTokenInformation(token: string, retries = 3) {
```

Should this be string?